UnitTests/AssigneeManagerTests/AssignUserToTicketTest.cs

```diff
@@ -0,0 +1,62 @@
+﻿using InternshipProject_2.Manager;
+using InternshipProject_2.Models;
+using RequestResponseModels.Assignee.Request;
+using RequestResponseModels.Assignee.Response;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTests.AssigneeManagerTests
+{
+    [TestClass]
+    public class AssignUserToTicketTest
+    {
+        private AssigneeManager _assigneeManager;
+        private Project2Context _project2Context;
+
+        [TestInitialize]
+        public void Setup()
+        {
+            _assigneeManager = new AssigneeManager();
+            _project2Context = new Project2Context();
+        }
+
+        [TestMethod]
+        public async Task AssignUserToTicket_ValidRequest_ReturnsSuccessResponse()
```

Comment:
You should differentiate the arrange, act, and asset sections.
![image](https://github.com/DobocanDarius/InternshipProject-2/assets/30144892/ce1d2d7c-ed4c-43b8-b2f1-02c2dd22d960)